libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationView.kt

```diff
@@ -102,19 +103,19 @@ class NavigationView @JvmOverloads constructor(
 
     init {
         keepScreenOn = true
+        val options = NavigationOptions.Builder(context)
+            .accessToken(accessToken)
+            .build()
 
         SharedApp.setup(context.applicationContext as Application)
         if (!MapboxNavigationApp.isSetup()) {
-            MapboxNavigationApp.setup(
-                NavigationOptions.Builder(context)
-                    .accessToken(accessToken)
-                    .build()
-            )
+            MapboxNavigationApp.setup(options)
         }
 
         MapboxNavigationApp.attach(this)
 
         attachCreated(
+            CustomEventComponent(navigationOptions = options),
```

Comment:
We can get the `navigationOptions` from `MapboxNavigation` instance in `onAttached` and `onDetached`. No need to pass it here as an arg.

Furthermore, if we update `mapboxNavigation.postCustomEvent` method as [described here](https://github.com/mapbox/mapbox-navigation-android/pull/5947/files#r909693108), we won't need `navigationOptions` at all.